### PR TITLE
testcne: update meson with metrics test

### PR DESCRIPTION
**Pull request by sushmasi**
Signed-off-by: Sushma Sitaram <sushma.sitaram@intel.com>

### CODE REVIEW

**Comment by jeffreybshaw** [comment]
This is failing because we do not run the tests as root, which is required to open the uds socket used by the metrics library.
One thing we could try is to create a new group of tests, i.e. "sudo_test_names" that run with "sudo meson test -C builddir --suite su-short" where we run the tests that require sudo.

**Review by KeithWiles — APPROVED** [review]

File: test/testcne/meson.build [hunk 1]
```diff
@@ -128,6 +128,7 @@ test_names = [
     'log',
     'mbuf',
     'mempool',
+    'metrics',
```
Comment (jeffreybshaw) [inline test/testcne/meson.build:131]: Please do not do this. We don't run the meson tests as root, and this test willl fail if it is not run as root. See #75

**Review by jeffreybshaw — CHANGES_REQUESTED** [review]

**Comment by jeffreybshaw** [comment]
@sushmasi - can you close this PR?

**Comment by sushmasi** [comment]
Changes taken care by commit 8770080db17e8df576b33622651b2ef985fef5f4

**Pull request closed without merging**